etc/env/base/chain.toml

```diff
@@ -65,8 +65,8 @@ compute_overhead_part = 0.0
 # If it is 1, the pubdata limit per batch will have to cover the entire cost of closing the batch.
 pubdata_overhead_part = 1.0
 
-# The constant amount of L1 gas that is used as the overhead for the batch. It includes the price for batch verification, etc.
-batch_overhead_l1_gas = 800000
+# The constant L1 gas unit in vbyte required to finilize the L1 Batch. This cost includes the Pubdata, proof and verification inscriptions.
```

Comment:
With Pubdata inscription you mean L1BatchDAReference? Also what is exactly covered by "verification inscriptions"? And finally is this just an estimate or you actually checked the size in bytes for all insriptions?

Reply:
This cost covers the commit pubdata, the proof and the vote inscriptions (verification). This is a random estimation with buffer, we need to calculate the correct one when we test all the flow.